pandas_gbq/exceptions.py

```diff
@@ -35,3 +35,12 @@ class PerformanceWarning(RuntimeWarning):
     Such warnings can occur when dependencies for the requested feature
     aren't up-to-date.
     """
+
+
+class QueryTimeout(ValueError):
+    """
+    Raised when the query request exceeds the timeoutMs value specified in the
+    BigQuery configuration.
+    """
+
+    pass
```

Comment:
nit: should we keep the `pass` here? Other error classes do not have this.

Reply:
You're correct. `pass` is _not_ needed because there is a docstring. I'll remove.